spec/takeway_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "takeaway.rb"
+
+describe Takeaway do 
+  it "shows menu" do
+    expect(subject.dish_price).to eq @menu
+  end
+
+  it "selects dish" do
+    expect(subject.select_dish("Small Chips")).to eq ["Small Chips"]
```

Comment:
It may be beneficial to test that `subject.order` contains small chips, and move `subject.select_dish("Small Chips") to the line immediately above the `expect`